backup/manager.go

```diff
@@ -241,13 +249,66 @@ func (b *Manager) IsSafeToRunNode(nodeID string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
 	for _, s := range snapshots {
 		if s.NodeID == nodeID && s.BackupID != "" && backupID != s.BackupID {
 			b.log.Errorf("remote restore was found for node %v.", nodeID)
 			b.log.Errorf("current backupID=%v, remote backupID-%v", backupID, s.BackupID)
 			return false, nil
 		}
 	}
+
+	return true, nil
+}
+
+// ensureLatestBackup checks whenever a given local state is newer than our snapshot.
+// if there is no local state, it returns true.
+func (b *Manager) EnsureLatestBackup(nodeID string) (bool, error) {
+
+	provider := b.GetProvider()
+	if provider == nil {
+		return true, ErrorNoProvider
```

Comment:
Why do we return error here? It doesn't look like an error case.

Reply:
True, this is not an error case return no error.